Tests/swift-sdk-swift-tests/InAppTests.swift

```diff
@@ -9,19 +9,10 @@ import XCTest
 @testable import IterableSDK
 
 class InAppTests: XCTestCase {
-
-    override func setUp() {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
-    }
-
-    override func tearDown() {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
-    }
-
     func testAutoShowInAppSingle() {
         let expectation1 = expectation(description: "testAutoShowInAppSingle")
         let expectation2 = expectation(description: "Count decrements after showing")
-
+        
```

Comment:
What is the difference here? Are we adding four spaces to the beginning of empty line here?